ros/src/computing/perception/detection/packages/lidar_tracker/nodes/imm_ukf_pda_tracker/src/imm_ukf_pda_tracker.cpp

```diff
@@ -0,0 +1,868 @@
+#include "ukf.h"
+#include "imm_ukf_pda_tracker.h"
+
+
+using std::cout;
+using std::endl;
+
+namespace States
+{
+  enum TrackingState: int
+  {
+     Die = 0,
+     Init = 1,
+     Stable = 4,
+     Lost = 10,
```

Comment:
these are just state names, so please separate count configuration from here.
in addition, I suggest creating count parameters.